src/AVFoundation/AVCaptureSynchronizedDataCollection.cs

```diff
@@ -0,0 +1,37 @@
+﻿//
+// AVCaptureSynchronizedDataCollection.cs
+//
+// Authors:
+//	Alex Soto  <alexsoto@microsoft.com>
+//
+// Copyright 2017 Xamarin Inc. All rights reserved.
+//
+
+#if IOS || MONOMAC
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using XamCore.Foundation;
+
+namespace XamCore.AVFoundation {
+	public partial class AVCaptureSynchronizedDataCollection { //: IEnumerable<AVCaptureOutput>
+		public AVCaptureSynchronizedData this [AVCaptureOutput captureOutput] {
+			get {
+				return GetSynchronizedData (captureOutput);
+			}
+		}
+
+		// TODO: Enable IEnumerable/NSFastEnumerator once radar://34641736 is fixed
+		//https://trello.com/c/iFtsFSWs
```

Comment:
@dalexsoto Any info about this rdar?

Reply:
Yep they said it was fixed in iOS 11.2  but since we are not touching that until next week I have not validated it